docs/MANAGED.md

```diff
@@ -1,56 +1,72 @@
-# Full managed resources
+# Resource Management in keycloak-config-cli
```

Comment:
hi @Motouom  please aligned the tag with ## and ### for subsections, also please just keep the tag 1 as Resource  management